src/dmd/common/timetrace.d

```diff
@@ -0,0 +1,438 @@
+/**
+
+Compilation time tracing, --ftime-trace.
+The time trace profile is output in the Chrome Trace Event Format, described
+here: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
```

Comment:
Can you add a comment here that the output is "optimized" for viewing with Tracy, and it should stay like that. Chrome's viewer is essentially unusable. https://github.com/wolfpld/tracy